test/inductor/test_distributed_patterns.py

```diff
@@ -16,6 +16,8 @@
 
 
 def init_fake_distributed(device="cpu"):
+    import torch.distributed.fsdp
```

Comment:
Can we add a test for this? Like, start a process that only imports dynamo, and make sure that it doesn't also import fsdp.

Reply:
(example: https://github.com/pytorch/pytorch/blob/2e5d95a0828060f816251671e8e59f2680f9f9be/test/test_cuda.py#L4809)

Reply:
Yes, thanks for the pointer. Will try out.